modules/basic/ds/arrow_utils.cc

```diff
@@ -77,7 +77,7 @@ std::shared_ptr<Column> CreateColumn(
     return std::make_shared<DoubleColumn>(chunked_array, chunk_size);
   } else if (type == arrow::large_utf8()) {
     return std::make_shared<StringColumn>(chunked_array, chunk_size);
-  } else if (type == arrow::binary()) {
+  } else if (type == arrow::large_binary()) {
```

Comment:
Could we keep the both case here?

Reply:
StringColumn's ArrayType is declared like `ConvertToArrowType<std::string>`.

Reply:
I see. Thanks for the clearification.